src/components/WebinarPageContent/SectionDetails.tsx

```diff
@@ -11,111 +11,275 @@ export const SectionDetailsUnStyled = ({ className }: SectionDetailsProps) => {
   return (
     <section className={`${className}`}>
       <div className="container">
-        <div className="row justify-content-center mb-4">
+        <div className="row mb-4">
           <div className="col-auto">
             <h2>Tech Webinar Details</h2>
           </div>
         </div>
         <div className="row">
-          <div className="col-12 col-lg-4 mb-4">
+          <div className="col-12 col-md-6 col-lg-4 mb-4">
             <div className="webinar">
-              <h6>Webinar 1:</h6>
-              <h3>TradeTrust Overview</h3>
-              <h5>15 Jul 2020, 10 – 11am</h5>
-              <p>
-                This non-technical session helps provide a foundational and critical understanding of TradeTrust as a
-                digital utility as well as the mental framing necessary as a pre-requisite for subsequent webinars.
-              </p>
+              <div className="row align-items-center">
+                <div className="col-auto">
+                  <h2 className="mb-0">1</h2>
+                </div>
+                <div className="col">
+                  <h5 className="mb-0">TradeTrust Overview</h5>
+                </div>
+                <div className="col-3 col-sm-2 col-md-3">
+                  <img className="img-fluid" src="/static/images/webinar/icon-01.svg" alt="" />
+                </div>
+              </div>
+              <div className="row align-items-center bg-blue">
+                <div className="col-auto">
+                  <TagSolidOrange className="mb-0 text-uppercase">Intro</TagSolidOrange>
+                </div>
+                <div className="col">
+                  <h6 className="mb-0">
+                    15 Jul 2020
+                    <br />
+                    10.00 – 11.00am
+                  </h6>
+                </div>
+              </div>
+              <div className="row">
+                <div className="col-12">
+                  <p>
+                    This non-technical session helps provide a foundational and critical understanding of TradeTrust as
+                    a digital utility as well as the mental framing necessary as a pre-requsite for subsequent webinars.
+                  </p>
+                </div>
+              </div>
             </div>
           </div>
```

Comment:
Each of these look like a repeatable block, do you want to have it rendered from data instead?

Reply:
yup sure